WordPress/src/main/res/values/strings.xml

```diff
@@ -800,8 +800,8 @@
     <string name="reader_btn_unfollow">Following</string>
 
     <!-- EditText hints -->
-    <string name="reader_hint_comment_on_post">Add your comment</string>
-    <string name="reader_hint_comment_on_comment">Reply to comment</string>
+    <string name="reader_hint_comment_on_post">Reply to post...</string>
+    <string name="reader_hint_comment_on_comment">Reply to comment...</string>
```

Comment:
The three dots here should probable be switched to the ellipsis character, `…` (or \&#8230;).